drizzlepac/haputils/astrometric_utils.py

```diff
@@ -2104,13 +2104,15 @@ def build_focus_dict(singlefiles, prodfile, sigma=2.0):
     for f in singlefiles:
         imgarr = fits.getdata(f)
         imgarr[~full_sat_mask] = 0
+        imgarr = np.nan_to_num(imgarr, 0)  # Insure, on Linux, all 'nan' values are removed
```

Comment:
Question as this always throws me off.  The signature for nan_to_num is 
numpy.nan_to_num(x, copy=True, nan=0.0, posinf=None, neginf=None).  So does your "0" map to copy as False?  Is this what you intend?  I have the same question for line 2115.

Reply:
I updated the calls in [77449e2](https://github.com/spacetelescope/drizzlepac/pull/1255/commits/77449e27c494b9b14d9e49d7e5dae314c9803fe1) to that function to explicitly set 'nan=0' to be perfectly clear with how this function should be used in this instance.